.jest/__fixtures__/algorithmAquarius.ts

```diff
@@ -0,0 +1,91 @@
+import { Asset } from '@oceanprotocol/lib'
+
+export const algorithmAquarius: Asset = {
```

Comment:
if you introduce `algorithmAquarius` then the other ones can't be called `assetAquarius` anymore, as "asset" is the collection term for both, datasets & algorithms. So with a newly named `algorithmAquarius`, the other ones need to be renamed to `datasetAquarius`